service/history/mutableStateBuilder.go

```diff
@@ -75,6 +79,8 @@ type (
 		updateChildExecutionInfos []*persistence.ChildExecutionInfo
 		deleteChildExecutionInfo  *int64
 		continueAsNew             *persistence.CreateWorkflowExecutionRequest
+		newBufferedEvents         []*persistence.SerializedHistoryEventBatch
```

Comment:
Does this needs to be an array?  Every update should create a single batch of serialized event.

Reply:
this is to make unit test pass